f/frizzle/comapeo/tests/comapeo_observations_test.py

```diff
@@ -25,11 +51,23 @@ def test_script_e2e(comapeoserver, pg_database, tmp_path):
     ).exists()
 
     with psycopg2.connect(**pg_database) as conn:
-        # Survey responses from forest_expedition are written to a SQL Table
+        # Survey responses from forest_expedition are written to a SQL Table in expected format
         with conn.cursor() as cursor:
             cursor.execute("SELECT COUNT(*) FROM comapeo_forest_expedition")
             assert cursor.fetchone()[0] == 3
 
+            cursor.execute(
+                "SELECT column_name FROM information_schema.columns WHERE table_name = 'comapeo_forest_expedition'"
+            )
+            columns = [row[0] for row in cursor.fetchall()]
```

Comment:
```suggestion
            cursor.execute("SELECT * FROM comapeo_forest_expedition LIMIT 0")
            columns = [desc[0] for desc in cursor.description]
```

I want to get out of the habit of querying `information_schema`: it's a leaky abstraction.